src/config.rs

```diff
@@ -46,3 +46,16 @@ pub fn zo_maxage() -> Result<Rank> {
         None => Ok(1000.0),
     }
 }
+
+pub fn zo_resolve_symlinks() -> bool {
+    match env::var_os("_ZO_RESOLVE_SYMLINKS") {
+        Some(resolve_osstr) => {
+            let resolve_str = resolve_osstr.to_str();
+            match resolve_str.unwrap_or("0").to_lowercase().as_ref() {
+                "1" => true,
+                _ => false,
+            }
+        }
+        None => false,
+    }
```

Comment:
This can be simplified:

```rust
match env::var_os("_ZO_RESOLVE_SYMLINKS") {
    Some(var) => var == "1",
    None => false,
}
```

Reply:
Nice catch, I was pretty stupid on this one.